src/model/otr_model.rs

```diff
@@ -263,82 +260,56 @@ impl OtrModel {
                     .expect("Player rating should exist");
                 (
                     player_id,
-                    Self::calc_rating_a(&ratings, current.rating, current.volatility, total_games)
-                )
-            })
-            .collect()
-    }
-
-    /// Calculates the penalized rating (Method B) for all players in a match.
-    ///
-    /// Method B uses the actual ratings calculated with missed games counted as losses,
-    /// providing a more punitive rating change for partially played matches.
-    fn calc_b(&self, rating_map: HashMap<i32, Vec<Rating>>, match_: &Match) -> HashMap<i32, Rating> {
-        let total_games = match_.games.len();
-        rating_map
-            .into_iter()
-            .map(|(player_id, ratings)| (player_id, Self::calc_rating_b(&ratings, total_games)))
-            .collect()
-    }
-
-    /// Combines Method A and B ratings using weighted average.
-    ///
-    /// The final rating is calculated as:
-    /// - Rating = (WEIGHT_A × Method A) + (WEIGHT_B × Method B)
-    /// - Volatility = √(WEIGHT_A × σ²_A + WEIGHT_B × σ²_B)
-    ///
-    /// Ensures the final rating stays within system bounds:
-    /// - Rating ≥ ABSOLUTE_RATING_FLOOR
-    /// - Volatility ≤ DEFAULT_VOLATILITY
-    fn calc_weighted_rating(&self, map_a: &HashMap<i32, Rating>, map_b: &HashMap<i32, Rating>) -> HashMap<i32, Rating> {
-        map_a
-            .keys()
-            .map(|&player_id| {
-                let result_a = map_a.get(&player_id).expect("Player should have Method A rating");
-                let result_b = map_b.get(&player_id).expect("Player should have Method B rating");
-
-                let rating = WEIGHT_A * result_a.mu + WEIGHT_B * result_b.mu;
-                let volatility = (WEIGHT_A * result_a.sigma.powf(2.0) + WEIGHT_B * result_b.sigma.powf(2.0)).sqrt();
-
-                (
-                    player_id,
-                    Rating {
-                        mu: rating.max(ABSOLUTE_RATING_FLOOR),
-                        sigma: volatility.min(DEFAULT_VOLATILITY)
-                    }
+                    Self::calc_weighted_rating(
+                        &ratings,
+                        &rating_map_b[&player_id],
+                        current.rating,
+                        current.volatility,
+                        total_games
+                    )
                 )
             })
             .collect()
     }
 
-    /// Calculates Method A rating for a player.
-    fn calc_rating_a(ratings: &[Rating], current_rating: f64, current_volatility: f64, total_games: usize) -> Rating {
-        let played_games = ratings.len();
-        let unplayed_games = total_games - played_games;
-
-        let rating_sum: f64 = ratings.iter().map(|r| r.mu).sum();
-        let rating = (rating_sum + current_rating * unplayed_games as f64) / total_games as f64;
+    /// Calculate a new weighted, game-corrected rating for a player given tentative "A" and "B"
```

Comment:
The use of 'tentative' is a bit odd, I would like to see this elaborated upon more. Touch on why we are doing this or link to an up-to-date docs page which explains this.